Buildings/Examples/VAVReheat/BaseClasses/VAVReheatBox.mo

```diff
@@ -72,7 +72,7 @@ model VAVReheatBox "Supply box of a VAV system with a hot water reheat coil"
     "Supply Air Volumetric Flow Rate"
     annotation (Placement(transformation(extent={{100,70},{120,90}}),
         iconTransformation(extent={{100,70},{120,90}})));
-  Buildings.Fluid.Actuators.Dampers.PressureIndependent vav(
+  Buildings.Fluid.Actuators.Dampers.Exponential vav(
```

Comment:
If we don't address the issue with `have_preIndDam` in this PR then this modification must be undone.
Note that

- The zone temperature setpoint is no longer met in the examples with this change.
- `Buildings.Examples.VAVReheat.BaseClasses.Controls.RoomVAV` which is used for the ASHRAE2006 example does not include a pressure independent *control logic*, so `Dampers.PressureIndependent` is needed there.
- However, I noticed that on the master branch, `have_preIndDam` is left at its default value `false` for the Guideline36 example, despite the model using `Dampers.PressureIndependent`. This is a mistake that should be corrected (or that may become obsolete after refactoring to remove `have_preIndDam` completely). But again, not through this PR I believe.

Reply:
@AntoineGautier Thanks for catching it. I have reverted the changes and Rerun the unit test. There is no reference results change.